common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2489,11 +2489,6 @@
           "Referencing lateral column alias <lca> in the aggregate query both with window expressions and with having clause. Please rewrite the aggregate query by removing the having clause or removing lateral alias reference in the SELECT list."
         ]
       },
-      "LATERAL_COLUMN_ALIAS_IN_GROUP_BY" : {
-        "message" : [
-          "Referencing a lateral column alias via GROUP BY alias/ALL is not supported yet."
```

Comment:
Could you also remove these lines:
https://github.com/apache/spark/blob/3f5d24f3e4895f3b2f97c13cbe516f900ce44d72/docs/sql-error-conditions-unsupported-feature-error-class.md#L68-L70

Reply:
Done